src/main/java/Duke.java

```diff
@@ -1,10 +1,67 @@
+package src.main.java;
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int MAXIMUM_TASK = 100;
+
     public static void main(String[] args) {
+        greetUser();
+        String userInput = "";
+        Task[] userInputList = new Task[MAXIMUM_TASK];
+        do {
+            userInput = scanInput();
+            switch(userInput) {
```

Comment:
Shouldn't the switch case be indented so that case statements are on the same indentation level as the switch statement? [(Link to the coding standard for reference)](https://nus-cs2113-ay1920s2.github.io/website/coding-standards/java/basic.html)

You can configure IntelliJ to automatically indent to follow this behaviour.